app/validators/blocks/list_collector_validator.py

```diff
@@ -21,14 +21,11 @@ class ListCollectorValidator(BlockValidator, ValidateListCollectorQuestionsMixin
     NO_RADIO_FOR_LIST_COLLECTOR_REMOVE = (
         "The list collector remove block does not contain a Radio answer type"
     )
-    LIST_COLLECTOR_ADD_EDIT_IDS_DONT_MATCH = (
-        "The list collector block contains an add block and edit block"
-        " with different answer ids"
-    )
-    NON_UNIQUE_ANSWER_ID_FOR_LIST_COLLECTOR_ADD = (
-        "Multiple list collectors populate a list using different "
-        "answer_ids in the add block"
+    LIST_COLLECTOR_FOR_SUPPLEMENTARY_LIST_IS_INVALID = (
+        "List collectors cannot be for a list which comes from supplementary data"
```

Comment:
For clarity, this should be either `Non content list collectors ...` or `Only ListCollectorContent types can be used for a list which comes from supplementary data`